hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/PrefixManagerImpl.java

```diff
@@ -176,9 +177,10 @@ public boolean setAcl(OzoneObj obj, List<OzoneAcl> acls) throws IOException {
       OmPrefixInfo prefixInfo =
           metadataManager.getPrefixTable().get(prefixPath);
 
-      OMPrefixAclOpResult omPrefixAclOpResult = setAcl(obj, acls, prefixInfo);
+      OMPrefixAclOpResult omPrefixAclOpResult = setAcl(obj, acls, prefixInfo,
+          0L);
```

Comment:
Dumb q. - why pass 0 here?

Reply:
This code path was used for the non-ratis case before the two code paths were merged. This should go away after code cleanup. 
@bharatviswa504 please correct me if I am wrong.

Reply:
You're right.